src/components/Header/HeaderContainer.js

```diff
@@ -1,25 +1,21 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import axios from 'axios';
 import PropTypes from 'prop-types';
 
-import { MAIN_URL } from '../../utils/url-utils';
+import { authAPI } from '../../api/api';
 import { setAuthUserData } from '../../redux/actions/actionCreators';
 
 import Header from './Header';
 
 class HeaderContainer extends Component {
   componentDidMount() {
-    axios.get(`${MAIN_URL}auth/me`, {
-      withCredentials: true,
-    })
-      .then((response) => {
-        const { props } = this;
-        if (response.data.resultCode === 0) {
-          const { id, email, login } = response.data.data;
-          props.setAuthUserData(id, email, login);
-        }
-      });
+    authAPI.me().then((response) => {
+      const { props } = this;
+      if (response.data.resultCode === 0) {
+        const { id, email, login } = response.data.data;
```

Comment:
Try using destructuring for "response" to get the "data" and then get the "data" and "resultCode" out of it.

Reply:
I will get rid of this in the next PR using thunk